services/identity-service/src/kyc/kyc.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { BadRequestException, Body, Controller, Get, Param, Post } from '@nestjs/common';
```

Comment:
Do we need these APIS?

Reply:
no, these were written to enable a mock aadhar based login flow to create DIDs, and have been removed